src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,28 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
user can be null, better to render this component only when the user exists
```suggestion
        {post.user && <UserInfo user={post.user} />}
```